Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.yml

```diff
@@ -0,0 +1,421 @@
+      type: String
+    - contextPath: SecurityScorecard.Company.Factor.Issue.total_score_impact
+      description: Contribution of issue on overall score
+      type: Number
+    - contextPath: SecurityScorecard.Company.Factor.Issue.detail_url
+      description: URL to the details of the issue
+      type: String
+    - contextPath: SecurityScorecard.Company.Factor.total
+      description: Number of factors returned
+      type: Number
+  - name: securityscorecard-company-history-score-get
+    description: Retrieve company historical scores
+    arguments:
+    - name: domain
+      description: Company domain, e.g. `google.com`
+      required: true
+      default: false
+    - name: from
+      description: Initial date for historical data. Value should be in format `YYYY-MM-DD`
+      required: false
+      default: false
+    - name: to
+      description: Initial date for historical data. Value should be in format `YYYY-MM-DD`
+      default: false
+    - name: timing
+      description: Timing granularity. Acceptable values are `weekly` or `daily`
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - "daily"
+      - "weekly"
+    outputs:
+    - contextPath: SecurityScorecard.Company.History.domain
+      description: Company domain.
+      type: String
+    - contextPath: SecurityScorecard.Company.History.date
+      description: Score date.
+      type: Date
+    - contextPath: SecurityScorecard.Company.History.score
+      description: Company historical security score in numeric form (55-100)
+      type: Number
+  - name: securityscorecard-company-history-factor-score-get
+    description: Retrieve company historical factor scores
+    arguments:
+    - name: domain
+      description: Company domain, e.g. google.com
+      required: true
+      default: false
+    - name: from
+      description: Initial date for historical data. Value should be in format 'YYYY-MM-DD'
+      required: false
+      default: false
+    - name: to
+      description: Initial date for historical data. Value should be in format 'YYYY-MM-DD'
+      default: false
+    - name: timing
+      description: Timing granularity. Acceptable values are "daily" (default), "weekly"
+        or "monthly"
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - "daily"
+      - "weekly"
+      - "monthly"
+    outputs:
+    - contextPath: SecurityScorecard.Company.FactorHistory.domain
+      description: Company domain.
+      type: String
+    - contextPath: SecurityScorecard.Company.FactorHistory.date
+      description: Score date.
+      type: Date
+    - contextPath: SecurityScorecard.Company.FactorHistory.Factor.name
+      description: Factor name.
+      type: Number
+    - contextPath: SecurityScorecard.Company.FactorHistory.score
+      description: Company historical security score in numeric form (55-100)
+      type: Number
+  - name: securityscorecard-alert-grade-change-create
```

Comment:
Need to keep the original name because the API has two types of alerts:
- `grade`
- `score`

Their query params are different so need to have two separate commands for alert creation.